.github/workflows/ci_cd.yml

```diff
@@ -256,6 +256,7 @@ jobs:
   embedding-tests:
     name: Embedding testing and coverage
     runs-on: ubuntu-latest
+    timeout-minutes: 30
```

Comment:
@RobPasMue  Some embedding tests are going beyond 5 hrs and failing (but passing on re run). I thought this would free the runners. Currently on avg, each run takes around 4 mins(for each version). Let me know if we should add this?